pkg/hostpath/hostpath.go

```diff
@@ -52,11 +52,21 @@ type hostPathVolume struct {
 	VolPath string `json:"volPath"`
 }
 
+type hostPathSnapshot struct {
+	SnaName  string                  `json:"snaName"`
```

Comment:
You can remove all Sna prefixes since they are part of the hostPathSnapshot object. So, SnaID being just Id would be the id of the snapshot. and so one.